src/hooks/useLicensees/useLicensees.js

```diff
@@ -19,92 +19,129 @@ import { licenseeFields } from './licenseeFields'
  */
 
 export const useLicensees = (licenseeId) => {
-
-    let formColumns = 1
-
-    const softwareTable = useSoftware()
-
-    const licenseeTable = useDBTableFactory('licensee', licenseeId)
-
-    const data = useMemo(() => {
-        
-        if (licenseeTable.isLoading || licenseeTable.isError || licenseeTable.data === undefined) return []
-
-        return licenseeTable.data.map(item => {
-            return { ...item, software: item.__softwareConnection__?.map(software => software.__software__.id) }
-        })
-
-    }, [licenseeTable.data, licenseeTable.isError, licenseeTable.isLoading])
-
-    const assignedLicensesTable = useDBTableFactory('assigned-license')
-
-    const create = useCallback(async (props) => {
-        const { name = '', notes = '', software } = props
-
-        software = software.map((software) => {
-            return { id: software };
+  let formColumns = 1;
+
+  const softwareTable = useSoftware();
+
+  const licenseeTable = useDBTableFactory('licensee', licenseeId);
+
+  const data = useMemo(() => {
+    if (
+      licenseeTable.isLoading ||
+      licenseeTable.isError ||
```

Comment:
weird newline or carriage return here

Reply:
indeed, on my current branch, locally, it is all on one line